bedrock/firefox/templates/firefox/new/desktop/download.html

```diff
@@ -70,6 +70,17 @@
 
 {% set show_mr106_browser_promo = switch('download-firefox-mr106-promo', ['en-US', 'en-CA', 'en-GB', 'fr', 'de']) %}
 
+{% if outdated %}
```

Comment:
Question: should we only show this message if the locale has also translated the notification string?